src/olympia/migrations/1068-remove-categories-table-users.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE 'categories';
+DROP COLUMN 'name';
```

Comment:
In sql column and table names should be quoted with `` ` `` not ` ' `. 
Also, you need an empty line at the end of the file